tests/displays/cavity_display/backend/test_fault.py

```diff
@@ -12,36 +10,28 @@
 )
 from lcls_tools.common.data.archiver import ArchiverValue, ArchiveDataHandler
 
+from displays.cavity_display.backend.fault import FaultCounter
+
 archiver_value = ArchiverValue()
 get_data_at_time_mock = MagicMock(return_value={"PV": archiver_value})
 get_values_over_time_range_mock = MagicMock(
     return_value={"PV": ArchiveDataHandler([archiver_value])}
 )
 
+with mock.patch(
+    "lcls_tools.common.data.archiver.get_data_at_time", get_data_at_time_mock
```

Comment:
I think this is also taken care of in line 29?

Reply:
You're right! I thought it had something to do with the import Fault, but I just moved that import Fault statement and deleted the duplicate code and it seems fine still!